### PR TITLE
Added alcatel 5080X android browser

### PR DESCRIPTION
**Pull request by jasonf20**
Hey guys, 

Added a device which was missing from android-browser-4.0 for the user agent: `Mozilla/5.0 (Linux; U; Android 6.0; ru-ru; 5080X Build/MRA58K) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Mobile Safari/537.36`

I was not able to figure out how to run the unit tests, so please let me know how to do that :)

### CODE REVIEW

File: tests/issues/issue-1923.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<?php
+
+return [
+    'issue-new-A' => [
```
Comment (mimmi20) [inline tests/issues/issue-1923.php]: please rename the test to `issue-1923-A`.

Comment (jasonf20) [reply tests/issues/issue-1923.php]: Fixed

**Comment by mimmi20** [comment]
> I was not able to figure out how to run the unit tests

Please look into the `.travis.yml` file. There is a `integration tests` stage.

File: tests/issues/issue-1923.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<?php
+
+return [
+    'issue-1923-A' => [
+        'ua' => 'Mozilla/5.0 (Linux; U; Android 6.0; ru-ru; 5080X Build/MRA58K) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Mobile Safari/537.36',
+        'properties' => [
+            'Comment' => 'Yandex Browser 16.2',
```
Comment (mimmi20) [inline tests/issues/issue-1923.php]: Comment does not match to Browser. Should be `Android Browser 4.0`.

File: tests/issues/issue-1923.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            'Browser' => 'Android Browser',
+            'Browser_Type' => 'Browser',
+            'Browser_Bits' => '32',
+            'Browser_Maker' => 'Google Inc',
+            'Browser_Modus' => 'unknown',
+            'Version' => '16.2',
```
Comment (mimmi20) [inline tests/issues/issue-1923.php]: Version does not match. Should be `4.0`.

File: tests/issues/issue-1923.php (hunk 2 above)
Comment (jasonf20) [reply tests/issues/issue-1923.php]: That's why I wanted to run the tests :P Any way fixed both mismatches

**Comment by jasonf20** [comment]
I tried running the command in the integartion_testing (with the env vars set)
I get `Class 'DOMDocument' not found` I made sure to install php-dom. 

I have never used php before so I'm not sure what else to do. I'll just wait for the BI Build

**Comment by mimmi20** [comment]
> .. I made sure to install php-dom ...

On travis we use PHP with the `-n` option. If you have a normal installation of PHP you have to skip this option.

**Comment by jasonf20** [comment]
Alright, running thanks!

**Comment by jasonf20** [comment]
Thanks for the quick merge and the help :+1:

**Pull request merged**